Traceback (most recent call last):
  File "C:\JupyterLab\resources\jlab_server\envs\base2exp\lib\site-packages\jupyter_cache\executors\utils.py", line 51, in single_nb_execution
    executenb(
  File "C:\JupyterLab\resources\jlab_server\envs\base2exp\lib\site-packages\nbclient\client.py", line 1093, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "C:\JupyterLab\resources\jlab_server\envs\base2exp\lib\site-packages\nbclient\util.py", line 84, in wrapped
    return just_run(coro(*args, **kwargs))
  File "C:\JupyterLab\resources\jlab_server\envs\base2exp\lib\site-packages\nbclient\util.py", line 62, in just_run
    return loop.run_until_complete(coro)
  File "C:\JupyterLab\resources\jlab_server\envs\base2exp\lib\asyncio\base_events.py", line 616, in run_until_complete
    return future.result()
  File "C:\JupyterLab\resources\jlab_server\envs\base2exp\lib\site-packages\nbclient\client.py", line 559, in async_execute
    await self.async_execute_cell(
  File "C:\JupyterLab\resources\jlab_server\envs\base2exp\lib\site-packages\nbclient\client.py", line 854, in async_execute_cell
    self._check_raise_for_error(cell, exec_reply)
  File "C:\JupyterLab\resources\jlab_server\envs\base2exp\lib\site-packages\nbclient\client.py", line 756, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
# percentage calculator
print('''Mit diesem Rechner erfahren Sie kinderleicht, 
         wie viel Sie sparen k√∂nnen und wie hoch der Kaufbetrag ist.''')

# input
base = float(input("Geben Sie den Grundwert ein (Euro): "))
percentage = int(input("Geben Sie den Rabatt ein (%): "))

# calculate the percentage value and the remaining purchase amount
value = (base/100) * percentage
netto = base - value

# output
print(f"Sie sparen {value} Euro und m√ºssen noch {netto} Euro bezahlen.")
------------------

[1;31m---------------------------------------------------------------------------[0m
[1;31mStdinNotImplementedError[0m                  Traceback (most recent call last)
[1;32m~\AppData\Local\Temp/ipykernel_20120/1273869717.py[0m in [0;36m<module>[1;34m[0m
[0;32m      4[0m [1;33m[0m[0m
[0;32m      5[0m [1;31m# input[0m[1;33m[0m[1;33m[0m[1;33m[0m[0m
[1;32m----> 6[1;33m [0mbase[0m [1;33m=[0m [0mfloat[0m[1;33m([0m[0minput[0m[1;33m([0m[1;34m"Geben Sie den Grundwert ein (Euro): "[0m[1;33m)[0m[1;33m)[0m[1;33m[0m[1;33m[0m[0m
[0m[0;32m      7[0m [0mpercentage[0m [1;33m=[0m [0mint[0m[1;33m([0m[0minput[0m[1;33m([0m[1;34m"Geben Sie den Rabatt ein (%): "[0m[1;33m)[0m[1;33m)[0m[1;33m[0m[1;33m[0m[0m
[0;32m      8[0m [1;33m[0m[0m

[1;32mC:\JupyterLab\resources\jlab_server\envs\base2exp\lib\site-packages\ipykernel\kernelbase.py[0m in [0;36mraw_input[1;34m(self, prompt)[0m
[0;32m   1001[0m         """
[0;32m   1002[0m         [1;32mif[0m [1;32mnot[0m [0mself[0m[1;33m.[0m[0m_allow_stdin[0m[1;33m:[0m[1;33m[0m[1;33m[0m[0m
[1;32m-> 1003[1;33m             raise StdinNotImplementedError(
[0m[0;32m   1004[0m                 [1;34m"raw_input was called, but this frontend does not support input requests."[0m[1;33m[0m[1;33m[0m[0m
[0;32m   1005[0m             )

[1;31mStdinNotImplementedError[0m: raw_input was called, but this frontend does not support input requests.
StdinNotImplementedError: raw_input was called, but this frontend does not support input requests.

